widgetssdk/src/main/java/com/glia/widgets/helper/ContextExtensions.kt

```diff
@@ -44,3 +44,6 @@ internal fun Context.showToast(
 ) {
     Toast.makeText(this, message, duration).show()
 }
+
+internal val Activity.rootView: View
+    get() = findViewById(android.R.id.content) ?: window.decorView.findViewById(android.R.id.content)
```

Comment:
`Activity.getWindow` might also be `null` if the activity is not visual. So technically both options might return `null` but in practice, I never experienced that though some error log in this case would be nice notification if it happens for clients. Also `findViewById` is a slow method so I would put it as second.
```suggestion
    get() = window?.decorView?.rootView ?: findViewById(android.R.id.content)?.rootView
```